iceoryx_binding_c/test/moduletests/test_server.cpp

```diff
@@ -465,7 +465,7 @@ TEST_F(iox_server_test, SendWorks)
     ASSERT_THAT(payload, Ne(nullptr));
     *static_cast<int64_t*>(payload) = 42424242;
 
-    iox_server_send(sut, payload);
+    EXPECT_THAT(iox_server_send(sut, payload), Eq(ServerSendResult_SUCCESS));
```

Comment:
Could you please add at least another test where `iox_server_send` fails so that we can verify that the return value is not always `SUCCESS`.